CHANGELOG.md

```diff
@@ -50,6 +50,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
    * Deprecated `pysat.Instrument._filter_netcdf4_metadata` and replaced it
      with `pysat.utils.io.filter_netcdf4_metadata`.
    * Changed `fname` from a kwarg to an arg in `pysat.Instruments.to_netcdf4`
+   * Deprecated support for 2D pandas datasets
+   * Deprecated `pysat.instruments.pysat_testing2d`
```

Comment:
These should be combined.

Reply:
I used two lines since different actions produce the deprecation warning

Reply:
Yeah, but when I add multiple methods to a class I don't add a line for each class.  I think it would be easier to read the log if similar things are grouped together, and these are both being deprecated for the same reason.

Reply:
One is in the generalized instrument class, the other is the instrument module.  Two different files.

Reply:
I completely disagree with you, because they both support 2D pandas datasets, but am not going to die on this hill.